elegant/gui/general_pose_annotator.py

```diff
@@ -0,0 +1,91 @@
+import freeimage
+import pathlib
+import pickle
+from PyQt5 import Qt
+import numpy
+from functools import partial
+
+from ris_widget import ris_widget
+from ris_widget import shared_resources
+
+from elegant.gui import pose_annotation
+from elegant import worm_widths
+from elegant import worm_spline
+
+from zplib.image import colorize
+
+class GeneralPoseAnnotator():
+    def __init__(self, rw):
+        self.ris_widget = rw
+        width_estimator = worm_widths.WidthEstimator.from_default_widths(pixels_per_micron=1/1.3)
+        self.pose_annotator = pose_annotation.PoseAnnotation(self.ris_widget, width_estimator=width_estimator)
+        self.ris_widget.add_annotator([self.pose_annotator])
+
+        
+        load_data = Qt.QPushButton('Load')
+        load_data.clicked.connect(self.load_metadata)
+        save_data = Qt.QPushButton('Save')
+        save_data.clicked.connect(self.save_annotations)
```

Comment:
Consistency in naming always helps in understanding code when you or someone else comes back to it some months later. Choose either "metadata" or "annotations" so that the load/save functions don't use different nouns...